frontend/web/components/pages/ChangeRequestPage.js

```diff
@@ -652,102 +650,97 @@ const ChangeRequestsPage = class extends Component {
+                                      newEnabled === oldEnabled ? 0.25 : 1,
+                                  }}
+                                />
+                              </Row>
+                              <Row className='list-item change-request-item px-4 align-items-start'>
+                                <div
+                                  className='font-weight-medium mr-3'
+                                  style={{ width: labelWidth }}
+                                >
+                                  Value:
+                                </div>
+                                <Flex
+                                  style={{
+                                    opacity: oldValue === newValue ? 0.25 : 1,
+                                  }}
+                                >
+                                  <label>Value</label>
+                                  <ValueEditor
+                                    value={newValue}
+                                    className={classnames('code-medium', {
+                                      'change-request-updated-value':
+                                        oldValue !== newValue,
+                                    })}
                                   />
-                                </Row>
-                                <Row className='list-item change-request-item px-4 align-items-start'>
-                                  <div
-                                    className='font-weight-medium mr-3'
-                                    style={{ width: labelWidth }}
-                                  >
-                                    Value:
-                                  </div>
-                                  <Flex
-                                    style={{
-                                      opacity: oldValue === newValue ? 0.25 : 1,
-                                    }}
-                                  >
-                                    <label>Value</label>
-                                    <ValueEditor
-                                      value={newValue}
-                                      className={classnames('code-medium', {
-                                        'change-request-updated-value':
-                                          oldValue !== newValue,
-                                      })}
-                                    />
-                                  </Flex>
-                                </Row>
+                                </Flex>
+                              </Row>
 
-                                {isMv &&
-                                  mvData.map((v, i) => (
-                                    <Row
-                                      key={i}
-                                      className='list-item px-4 align-start change-request-item'
+                              {isMv &&
+                                mvData.map((v, i) => (
```

Comment:
I understanding you're just dealing with existing code and changing the tabbing, but since it is involved with the PR I'll just mention that the name `isMv` and `mvData` and `map((v, i) => ...` make this section sorta hard to understand.

Reply:
Sure, this is essentially showing differences in change request on multivariate flag variations.